datashader/bundling.py

```diff
@@ -0,0 +1,266 @@
+from __future__ import absolute_import, division, print_function
+
+from math import ceil
+
+from dask import compute, delayed
+from pandas import DataFrame
+from skimage.filters import gaussian, sobel_h, sobel_v
+
+import numba as nb
+import numpy as np
+import pandas as pd
+
+from .utils import ngjit
+
+
+ACCURACY = 500
+
+MIN_SEG = 0.008
+MAX_SEG = 0.016
+TENSION = 0.3
+
+
+@ngjit
+def distance_between(a, b):
+    return (((a[0] - b[0]) ** 2) + ((a[1] - b[1]) ** 2))**(0.5)
+
+
+@nb.jit
+def resample_segment(segments, new_segments, min_segment_length=MIN_SEG, max_segment_length=MAX_SEG):
+    next_point = np.array([0.0, 0.0])
```

Comment:
Please add function docstrings to any functions you've studied long enough to know what to put in them.